classes/test.php

```diff
@@ -1462,20 +1448,40 @@ public function errorHandler($errno, $errstr, $errfile, $errline)
         return $doNotCallDefaultErrorHandler;
     }
 
-    public function setUp()
+    private function callSetUp()
```

Comment:
These new `call*` methods must be protected to be overriden in the `phpunit-extension`. See my comment to have more explanations.